src/EditorFeatures/VisualBasicTest/Diagnostics/InsertMissingCast/InsertMissingCastTests.vb

```diff
@@ -1,309 +0,0 @@
-﻿' Licensed to the .NET Foundation under one or more agreements.
-' The .NET Foundation licenses this file to you under the MIT license.
-' See the LICENSE file in the project root for more information.
-
-Imports Microsoft.CodeAnalysis.CodeFixes
-Imports Microsoft.CodeAnalysis.Diagnostics
-Imports Microsoft.CodeAnalysis.VisualBasic.CodeFixes.InsertMissingCast
-
-Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics.InsertMissingCast
-    Public Class InsertMissingCastTests
-        Inherits AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest
-
-        Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace) As (DiagnosticAnalyzer, CodeFixProvider)
-            Return (Nothing, New InsertMissingCastCodeFixProvider)
-        End Function
-
-        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsInsertMissingCast)>
-        Public Async Function TestPredefinedAssignment() As Task
```

Comment:
were all these tests preserved?

Reply:
yup they are included in the new test file